### PR TITLE
Add accessibilityElementsHidden prop

### PR DESCRIPTION
**Pull request by aputinski**
## Motivation

Allow iOS to have similar accessibility functionality to Android. This PR exposes the `accessibilityElementsHidden` property on iOS which is similar to Android's `importantForAccessibility="no-hide-descendants"`

## Test Plan

I didn't see any existing examples for testing native props being passed through, but I did add an example to the RNTester app. I've attached some screenshots using the Accessibility Inspector to verify the property was correctly passed through.

a
b

## Related PRs

I've updated the website with appropriate documentation.

https://github.com/facebook/react-native-website/pull/141

screen shot 2018-01-16 at 10 23 50
screen shot 2018-01-16 at 10 23 59

## Release Notes

[IOS] [FEATURE] [View] - Added accessibilityElementsHidden property

### CODE REVIEW

**Comment by shergin** [comment]
Well, this is really great.
Thank you for raising this topic! However, I think we have to land https://github.com/facebook/react-native/pull/10918 instead... finally.

@aputinski What do you think?

**Comment by aputinski** [comment]
Looks like that was closed and we're now discussing https://github.com/facebook/react-native/pull/11788. I don't have a strong preference, but it seems like there is quite a bit of confusion about trying to unify this behavior. Would it be possible to merge this (since it's non-breaking) and then continue discussing unifying this property down the road?

@shergin @omeid

File: .gitignore [hunk 1]
```diff
@@ -48,6 +48,7 @@ node_modules
 *.log
 .nvm
 /danger/node_modules/
+scripts/.packager.env
```
Comment (shergin) [inline .gitignore]: Why do we need this?

**Comment by shergin** [comment]
@aputinski Yeah, that's reasonable. Even if I am going to land that PR as well, it's "safer" to land yours first.

**Comment by omeid** [comment]
I think landing this is a mistake without understanding the android part. The reason I abandoned my initial PR was because no one from facebook actually showed interested and were willing to consider a practical solution.

I am still interested to see this feature landed so I don't have to maintain a fork that has some decent a18y.

File: .gitignore (hunk 1 above)
Comment (aputinski) [reply .gitignore]: When I was running the build locally, that file was generated and I didn't want to commit any build artifacts. It's unrelated to my change and I can remove it if necessary.

**Pull request closed without merging**